modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js

```diff
@@ -142,15 +142,50 @@ export default function propsTransformer({portletNamespace, ...otherProps}) {
 			onSelect: (selectedItems) => {
 				let redirectURL = itemData?.redirectURL;
 
+				const selectedItemsJSONArray = [];
+
 				selectedItems.forEach((item) => {
-					redirectURL = addParams(
-						`${portletNamespace}contentDashboardItemSubtypePayload=${JSON.stringify(
-							item
-						)}`,
-						redirectURL
-					);
+					let entryClassNameItem;
+
+					if (item.entryClassName && item.className) {
+						entryClassNameItem = selectedItemsJSONArray.find(
+							(object) =>
+								object.entryClassName === item.entryClassName
+						);
+					}
+					else if (item.entryClassName) {
+						entryClassNameItem = selectedItemsJSONArray.find(
+							(object) =>
+								object.entryClassName === item.entryClassName
+						);
+					}
+
+					if (entryClassNameItem) {
+						entryClassNameItem.classPKs.push(item.classPK);
+					}
+					else {
+						if (item.className) {
+							selectedItemsJSONArray.push({
+								className: item.className,
+								classPKs: [item.classPK],
+								entryClassName: item.entryClassName,
+							});
+						}
+						else {
```

Comment:
Is this condition ever reached? I mean, could it be that an item does not have a `className/classPK`, but just the `entryClassName`?

Reply:
Yes it is possible to have an item without className. So, yes this condition can be reached.

Reply:
Ok then, thanks!